umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/iso19115_2/spatial.clj

```diff
@@ -43,6 +44,49 @@
                 (:EastBoundingCoordinate  rect)
                 (:SouthBoundingCoordinate rect))))))
 
+(defn- remove-whitespace
+  [value]
+  (when value
+    (-> (clojure.string/split value #"\s")
+        clojure.string/join)))
```

Comment:
This feels like a function that we would have in a utility namespace. If not, maybe move it there?